BUG: cannot run without sudo

If packagelock is ran without sudo, inside a directory that the execution user doesnt have permissions on, we get a very cryptic bug!
Stack Trace:
```go
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x793d7b]

goroutine 1 [running]:
go.uber.org/zap.(*Logger).clone(...)
	/home/nick/go/pkg/mod/go.uber.org/zap@v1.27.0/logger.go:318
go.uber.org/zap.(*Logger).Sugar(...)
	/home/nick/go/pkg/mod/go.uber.org/zap@v1.27.0/logger.go:147
packagelock/logger.InitLogger()
	/home/nick/Work/programming/Golang/packagelock/logger/logger.go:48 +0x1db
main.init.0()
	/home/nick/Work/programming/Golang/packagelock/main.go:219 +0x10a
```

--- a/logger/logger.go
+++ b/logger/logger.go
@@ -1,6 +1,7 @@
 package logger
 
 import (
+	"fmt"
 	"os"
 
 	"github.com/k0kubun/pp"
@@ -41,7 +42,15 @@ func InitLogger() (*zap.SugaredLogger, error) {
 	// Build the logger
 	logger, err := loggerConfig.Build()
 	if err != nil {
-		// Handle logger initialization error (no panic)
+		sudoErrorMessage := `
+		We cannot create a Logger. This oftenly fails when the User running PackageLock
+		doesnt has Read & Write Permissions on the Directory PackageLock is running in.
+
+		You can change the Permissions with '$ (sudo) chmod ---' or run PackageLock with 'sudo'.
+		🚀
+		`
+		fmt.Println(sudoErrorMessage)
+		pp.Printf("Cannot init Logger, got: %s")
 		logger = nil
 	}
 	// Return the initialized logger and any error that occurred

